### DISCUSSION TITLE
Add the Entity as a Card

### DISCUSSION THREAD

**Discussion by Judocrafter** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#139] [Q&A] [answered]
How can i add the entity camera.xiaomi_cloud_map_extractor now to my lovelace Dashboard?
Thank you for your help.
![image](https://user-images.githubusercontent.com/42749857/123045088-ca0bfc80-d3fa-11eb-867a-a09b66562a1f.png)

**Comment by PiotrMachowski** [comment, marked as answer]
* [Xiaomi Vacuum Map card](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card) (Viomi not supported at this moment)
* [Picture Entity Card](https://www.home-assistant.io/lovelace/picture-entity/)
* [Picture Glance Card](https://www.home-assistant.io/lovelace/picture-glance/)
